shard_connection.h

```diff
@@ -172,6 +174,8 @@ class shard_connection {
     struct bufferevent *m_bev;
     struct event_base* m_event_base;
 
+    bool limited;
```

Comment:
the only usage of this one is in fill_pipeline... why we can't use local variable instead of class's member?

Reply:
💯 agree. corrected in the latest commit.